### ISSUE TITLE
Filtering not accessible to screen reader users

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#289]
# What’s the issue?

The ability to filter by tags is not able to be used by keyboard users that have screen readers enabled. Users do not have the ability to add correct tags, the ability to remove tags, or to access all tags that are displayed in the auto-complete that appears.

# Why it matters

This has been identified as a critical issue as it is currently not usable or accessible to all users.

# Our recommendation

We would like to workshop with you in the debrief.

Our recommended pattern would be filters as checkboxes and a Filter button. The
filters could be hidden in an accordion and only expanded as needed.

**Comment by imperator99** [comment]
Resolved by https://github.com/govau/dta-gov-au/pull/299 and https://github.com/govau/dta-website-rebuild/pull/377.

**Comment by imperator99** [comment]
A suggestion has been made to leave the accordion open once something has been ticked so users can quickly identify which tags have been selected.

**Comment by imperator99** [comment]
Resolve by https://github.com/govau/dta-gov-au/pull/299, https://github.com/govau/dta-website-rebuild/pull/377 and https://github.com/govau/dta-gov-au/pull/333.
